src/main/scala/shine/DPIA/primitives/intermediate/SlideSeqIValues.scala

```diff
@@ -38,9 +38,9 @@ object SlideSeqIValues {
           // use neighborhood
           k(rs.rd) `;`
           // rotate
-          MapSeqI(size - 1, dt1, dt2, load,
+          MapSeqI(unroll = true)(size - 1, dt1, dt2, load,
             Drop(1, size - 1, dt1, rs.rd),
-            TakeAcc(size - 1, 1, dt2, rs.wr), unroll = true)
+            TakeAcc(size - 1, 1, dt2, rs.wr))
         ),
       arithexpr.arithmetic.RangeAdd(0, n, 1)))
     }))
```

Comment:
maybe this should become `RotateValuesI` for consistency.

Reply:
You mean renaming the file? Yes. We could do that.